From: Carsten Schoenert <c.schoenert@t-online.de>
Date: Sun, 3 Aug 2014 11:32:05 +0200
Subject: fix FTBFS on alpha

The jmalloc compiler fails on the alpha platform do missmatch of the
page size. This happen do incorrect settings for alpha
http://buildd.debian-ports.org/status/fetch.php?pkg=icedove&arch=alpha&ver=31.0~b2-1&stamp=1406501764

Adding the alpha platform with a correct define for 'pagesize_2pow' to the
defines section within mozilla/memory/mozjemalloc/jemalloc.c.

Patch based on suggestings for Bug #650151, namely the file
alpha-hardcode-pagesize.diff
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=650151
---
 mozilla/memory/mozjemalloc/jemalloc.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/mozilla/memory/mozjemalloc/jemalloc.c b/mozilla/memory/mozjemalloc/jemalloc.c
index 3be5790..3c93869 100644
--- a/mozilla/memory/mozjemalloc/jemalloc.c
+++ b/mozilla/memory/mozjemalloc/jemalloc.c
@@ -1103,6 +1103,8 @@ static const bool config_recycle = false;
 #define pagesize_2pow			((size_t) 13)
 #elif defined(__powerpc64__) || defined(__aarch64__)
 #define pagesize_2pow			((size_t) 16)
+#elif defined(__alpha__)
+#define pagesize_2pow			((size_t) 13)
 #else
 #define pagesize_2pow			((size_t) 12)
 #endif
